src/components/ngui-map.component.ts

```diff
@@ -136,7 +139,7 @@ export class NguiMapComponent implements OnChanges, OnDestroy, AfterViewInit, Af
       });
 
       // update map when input changes
-      debounceTime.call(this.inputChanges$, 1000)
+      debounceTime.call(this.inputChanges$, typeof this.config.inputDebounceTime === 'number' ? this.config.inputDebounceTime : 1000)
```

Comment:
no need to check the type, typescript will warn users if they pass a wrong value
```ts
this.config && this.config.inputDebounceTime ?
```